app/views/admin/shared/_tabnav_payment_method.html.haml

```diff
@@ -1,7 +1,8 @@
 %ul.nav.nav-tabs
   = tab t(".details"), facility_account_path(current_facility, @account), (secondary_tab == "details")
   -# TODO: use a feature flag here instead
-  - if @account.statements.any?
+  - if @account.statements.any? && !current_user.account_manager?
     = tab t(".statements"), facility_account_statements_path(current_facility, @account), (secondary_tab == "statements")
   = tab t(".members"), facility_account_members_path(current_facility, @account), (secondary_tab == "members")
-  = tab t(".orders"), facility_account_orders_path(current_facility, @account), (secondary_tab == "orders")
+  - unless current_user.account_manager?
```

Comment:
There is probably a more sensible thing to switch on, but I'm not sure quite what it would be.